patches/api/0327-Added-OP-level-API.patch

```diff
@@ -0,0 +1,75 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: MCMDEV <john-m.1@gmx.de>
+Date: Thu, 5 Aug 2021 17:00:29 +0200
+Subject: [PATCH] Added OP level API
+
+
+diff --git a/src/main/java/org/bukkit/command/MessageCommandSender.java b/src/main/java/org/bukkit/command/MessageCommandSender.java
+index a7ef1f51c2b96617a32e6e7b1723e8770ba8a6a8..d7484f5c7c14ce68e280d10950b99b027e7b6fac 100644
+--- a/src/main/java/org/bukkit/command/MessageCommandSender.java
++++ b/src/main/java/org/bukkit/command/MessageCommandSender.java
+@@ -60,6 +60,18 @@ public interface MessageCommandSender extends CommandSender {
+         throw new NotImplementedException();
+     }
+ 
++    //Paper start - OP level API
++    @Override
++    default int getOpLevel()    {
```

Comment:
What the fuck is this formatting

Reply:
I don't see the issue here right now

Reply:
Your opening `{` bracket is four spaces from the rest of the method signature.

Reply:
The spaces should be fixed now.

Reply:
Everything still seems to have two spaces in between the method signature and the opening {:
```java
public int getOperatorLevel(GameProfile profile)  {
```